app/views/content.js

```diff
@@ -13,21 +13,27 @@ const Content = Mn.LayoutView.extend({
         tracks: '.tracks',
     },
 
-    modelEvents: {
-        'change:currentPlaylist': 'switchPlaylist'
+    onBeforeShow() {
+        this.showChildView('header', new HeaderView({ model: this.model }));
+        this.showChildView('tracks', new TracksView({
+            collection: application.allTracks.get('tracks')
+        }));
+        this.setClass(application.appState.get('currentPlaylist'));
     },
 
-    onBeforeShow () {
-        this.showChildView('header', new HeaderView({ model: this.model }));
-        this.switchPlaylist(null, this.model.get('currentPlaylist'));
+    initialize() {
+         this.listenTo(application.appState,
+            'change:currentPlaylist',
+            (appState, currentPlaylist) => {
+                this.setClass(currentPlaylist);
+            }
+        );
     },
 
-    // only the router change the current playlist
-    switchPlaylist (appState, currentPlaylist) {
-        this.showChildView('tracks', new TracksView({
-            collection: currentPlaylist.get('tracks')
-        }));
-    }
+    setClass(currentPlaylist) {
+        let type = currentPlaylist.get('tracks').type;
+        this.$el.children('.tracks').removeClass().addClass('tracks ' + type);
```

Comment:
yup i know :)
But i need to remove the old class `type` but keep the 'tracks' class, i think this is the easiest method to do it ?